index.js

```diff
@@ -0,0 +1,274 @@
+/**
+ * Full fhirconvert function: validates, converts, then identifies variables.
+ * @returns converted fhirpath expression
+ */
+export function fhirconvert(str, vars, funs) {
```

Comment:
Why are "funs" a parameter?  Isn't the list of functions determined by your module?